pipeline_model_description.html

```diff
@@ -46,29 +45,35 @@ <h3 id='sourcebuffer1'>Flowing down from SourceBuffer 1</h3>
 then follows to an indicator of an open switch, which then continues to an
 indicator (green circle with an X) labeled Audio Device.</p>
 
-<p>The third Track Buffer also flows to a green box labeled Video Decoder,
+<p>The third Track Buffer also flows to a green box labeled Audio Decoder,
 which also flows to a switch, this time however indicated closed, which then
 also continues to the same indicator (green circle with an X) labeled Audio
 Device as the second Track Buffer.</p>
 
 
 
-
 <h3 id='sourcebuffer2'>Flowing down from SourceBuffer 2</h3>
 
 
 <p>Flowing down from SourceBuffer 2 is a mauve triangle with an arrow flowing
-to a red box labeled Video Decoder, which then flows to the same  Video Tag
-Display Region previously mentioned in SourceBuffer 1.</p>
+to a process indication labeled Track Buffer.<p>
+
+<p>This Track Buffer then flows to a red box labeled Video Decoder, which
+then flows to the same Video Tag Display Region previously mentioned in
+SourceBuffer 1.</p>
+
 
 
 <h3 id='sourcebuffer3'>Flowing down from SourceBuffer 3</h3>
 
 
 <p>Flowing down from SourceBuffer 3 is a rose triangle with an arrow flowing to
-a green box labeled Audio Decoder, which then flows through a closed switch,
-which then also continues to the same indicator (green circle with an X)
-labeled Audio Device previously mentioned in SourceBuffer 1</p>
+a process indication labeled Track Buffer.<p>
+
+<p>This Track Buffer then flows to a green box labeled Audio Decoder, which
+then flows through a closed switch, which then also continues to the same
+indicator (green circle with an X) labeled Audio Device previously mentioned in
```

Comment:
I think we could rephrase this to describe the purpose of "indicator (green circle with an X)". In a flow chart (I just looked this up...) the symbol is a "summing junction" which suggests that any enabled audio tracks are summed, i.e., mixed to form the output audio. This corresponds with definition of `AudioTrack`. 

Unless you prefer to do it, I could raise a PR with suggested text along those lines.

Reply:
Good catch. PR sounds good to me if you have time. I've filed an issue to ensure this gets done one way or the other :)
https://github.com/w3c/media-source/issues/314